api/services/execution/execution.proto

```diff
@@ -28,49 +28,76 @@ service ContainerService {
 }
 
 message CreateRequest {
-	string id = 1;
-	google.protobuf.Any spec = 2;
+	// ContainerID specifies the container to use for creating this task.
+	//
+	// The spec from the provided container id will be used to create the
+	// task associated with this container. Only one task can be run at a time
+	// per container.
+	//
+	// This should be created using the Containers service.
+	string container_id = 2;
+
+	// RootFS provides the pre-chroot mounts to perform in the shim before
+	// executing the container task.
+	//
+	// These are for mounts that cannot be performed in the user namespace.
+	// Typically, these mounts should be resolved from snapshots specified on
+	// the container object.
 	repeated containerd.v1.types.Mount rootfs = 3;
-	string runtime = 4;
+
 	string stdin = 5;
 	string stdout = 6;
 	string stderr = 7;
 	bool terminal = 8;
+
 	types.Descriptor checkpoint = 9;
 }
 
 message CreateResponse {
-	string id = 1;
-	uint32 pid = 2;
+	// TODO(stevvooe): We no longer have an id for a task since they are bound
+	// to a single container. Although, we should represent each new task with
+	// an ID so one can differentiate between each instance of a container
+	// running.
+	//
+	// Hence, we are leaving this here and reserving the field number in case
+	// we need to move in this direction.
+	// string id = 1;
+
+	string container_id = 2;
+	uint32 pid = 3;
 }
 
 message StartRequest {
-	string id = 1;
+	string container_id = 1;
```

Comment:
A container can only have a single active task. We still need to decide if we want to individually address them.